File: 140~149/ABC141/attack.c

#include<stdio.h>

int main(void){
    int n, q, a;
    long int k;
    scanf("%d""%ld""%d", &n, &k, &q);
    long int data[n];
    for(int i = 0; i < n; i++){
        data[i] = 0;
    }
    /*for(int i = 0; i < n; i++){
        printf("%ld \n", data[i]);
    }
    printf("\n");*/
    for(int i = 0; i < q; i++){
        scanf("%d", &a);
        data[a - 1] += 1;
    }
    for(int i = 0; i < n; i++){
        if(q - data[i] >= k){
            printf("No\n");
        }else{
            printf("Yes\n");
        }
    }
    return 0;
}

File: 140~149/ABC141/attack.py

n, k, q = map(int, input().split())
score = [k] * n
data = [0] * n

for i in range(q):
    a = int(input())
    data[a - 1] += 1

for i in range(n):
    if(q - data[i] >= k):
        print("No")
    else:
        print("Yes")
    
